core/WidgetsList.php

```diff
@@ -193,8 +255,12 @@ public static function getWidgetUniqueId($controllerName, $controllerAction, $cu
      * @param string $controllerAction The report's controller action method name.
      * @param array $customParameters Extra query parameters that should be sent while getting
      *                                this report.
+     * @param int $order The order hint
+     * @param string $widgetSubCategory The widget subcategory. This can be a translation token.
+     * @param int $order The order hint.
+     * @deprecated since Piwik 2.15, use `addWidget()` instead.
      */
-    public static function add($widgetCategory, $widgetName, $controllerName, $controllerAction, $customParameters = array())
+    public static function add($widgetCategory, $widgetName, $controllerName, $controllerAction, $customParameters = array(), $order = 99, $widgetSubCategory = '')
```

Comment:
This method will be removed in #7822 and we will only work on `WidgetConfig` instances